### DISCUSSION TITLE
How to build and use antd docs offline ?

### DISCUSSION THREAD

**Discussion by BobForCat** [ant-design/ant-design#30156] [Q&A]
Hi, 

I've just found https://github.com/ant-design/ant-design/discussions/28870 (antd docs locally) but I have something to ask about:

- is there a way to build and serve the docs rather that live-running from the source ? To respond faster maybe
- is there the way to handle the offline for icons ? A lot are stored and fetched from https://gw.alipayobjects.com/zos/ and so they aren't seen on the offline version, that isn't a problem for using the docs, just not nice

Thanks !

**Comment by yoyo837** [comment]
You mean to build static html?

**Reply by BobForCat** [reply]
Yep, like I'd do for a "classic" projet (https://create-react-app.dev/docs/deployment/)

 `npm run build` and `serve` or `npm run build` and copy `build` folder into a nginx server

**Reply by yoyo837** [reply]
I think it is possible, but I have not tried it, it can be built with `bisheng build`.

[bisheng cli](https://github.com/benjycui/bisheng#cli)

**Reply by afc163** [reply]
npm run site

**Comment by BobForCat** [comment]
To build static html and serve it: 

```bash
npm run site
serve -s _site
```
